cmd/kubectl/OWNERS

```diff
@@ -2,3 +2,6 @@ approvers:
 - sig-cli-maintainers
 reviewers:
 - sig-cli
+labels:
+- area/kubectl
+- sig/cli
```

Comment:
wow, this is so cool! I will add this for api-machinery repos as well. :tada: 

@cblecker @dixudx does this also apply to recursively to all  files under this `cmd/kubectl` or just the files under `cmd/kubectl` but not the nested ones?

Reply:
> does this also apply to recursively to all files under cmd/kubectl or just the files under cmd/kubectl but not the nested ones?

it applies recursively. :+1:

Reply:
awesome! :tada: